src/pudl/package_data/meta/xlsx_maps/eia861/tab_maps/operational_data_map_eia861.csv

```diff
@@ -32,4 +32,4 @@ revenue_from_credits_or_adjustments,61,56,28,28,28,28,28,27,27,27,27,27,27,27,27
 revenue_from_transmission,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,28,28,28,28,28,28,28,28,28
 revenue_from_other,62,57,29,29,29,29,29,28,28,28,28,29,29,29,29,29,29,29,29,29
 total,63,58,30,30,30,30,30,29,29,29,29,30,30,30,30,30,30,30,30,30
-data_type_o_=_observed_i_=_imputed,-1,-1,31,31,31,31,31,30,30,30,30,31,31,31,31,31,31,31,31,31
\ No newline at end of file
+data_type_o_=_observed_i_=_imputed,-1,-1,31,31,31,31,31,30,30,30,30,31,31,31,31,31,31,31,31,31

```

Comment:
Unrelated to this commit but, what is going on with this crazy column name? Does this actually work? Or do we need to make the name normalization function stricter? I thought it only resulted in alphanumerics + `_`? @swinter2011 @cmgosnell